src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/DraftApprenticeship.cs

```diff
@@ -95,13 +95,16 @@ public void Merge(DraftApprenticeshipDetails source, Party modifyingParty)
                     break;
             }
 
-            if(source.DeliveryModel == Types.DeliveryModel.PortableFlexiJob)
+            if (source.DeliveryModel == Types.DeliveryModel.PortableFlexiJob)
```

Comment:
I don't seem to be able to see any tests for this behaviour. But there might be as I haven't downloaded the branch

Reply:
Tests added to WhenDraftApprenticeshipIsUpdated